recipes/qwt/6.2.0/conanfile.py

```diff
@@ -0,0 +1,92 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, get
+
+required_conan_version = ">=1.50"
+
+class QwtConan(ConanFile):
+    name = "qwt"
+    license = "LGPL-2.1-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://qwt.sourceforge.io/"
+    topics = ("conan", "archive", "compression")
+    description = (
+        "The Qwt library contains GUI Components and utility classes which are primarily useful for programs "
+        "with a technical background. Beside a framework for 2D plots it provides scales, sliders, dials, compasses, "
+        "thermometers, wheels and knobs to control or display values, arrays, or ranges of type double."
+    )
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "plot": [True, False],
+        "widgets": [True, False],
+        "svg": [True, False],
+        "opengl": [True, False],
+        "designer": [True, False],
+        "polar": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "plot": False,
+        "widgets": False,
+        "svg": True,
+        "opengl": False,
+        "designer": False,
+        "polar": False,
+        "qt:qtsvg": True
+
+    }
+
+    def _patch_sources(self):
+        apply_conandata_patches(self)
+
+    def export_sources(self):
+        for p in self.conan_data.get("patches", {}).get(self.version, []):
+            copy(self, p["patch_file"], self.recipe_folder, self.export_sources_folder)
+
+    def build_requirements(self):
+        if self.settings.os == "Windows" and self.settings.compiler == "Visual Studio":
+            self.build_requires("jom/1.1.3")
+
```

Comment:
```suggestion
        self.tool_requires("qt/5.15.5") 

```
This is still needed, because MOC will be called during qwt compilation